ExtractMileage/Program.cs

```diff
@@ -109,10 +109,14 @@ static async Task Main(string[] args)
                 {
                     WriteXml(odometerReadings, fileName, utcNow);
                 }
-                else
+                else if (fileName.EndsWith(".csv", StringComparison.OrdinalIgnoreCase))
                 {
                     WriteCsv(odometerReadings, fileName, utcNow);
                 }
+                else 
+                {
+                    Console.WriteLine("Unknown file type");
```

Comment:
```suggestion
                    Console.WriteLine(" Unknown file type");
```

Reply:
just to be consistent with the rest of the Console.WriteLine commands in the file